hortum_mobile/lib/view/features/login/login_page.dart

```diff
@@ -0,0 +1,295 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:hortum_mobile/data/login_backend.dart';
+import 'package:hortum_mobile/view/features/introScreen_page.dart';
+
+import '../register/page/register_page.dart';
+
+class LoginPage extends StatefulWidget {
+  //LoginPage({Key key) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  bool _isObscure = true;
+  TextEditingController emailController = new TextEditingController();
+  TextEditingController passwordController = new TextEditingController();
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Form(
+        key: _formKey,
+        child: ListView(
+          children: [
+            SizedBox(
+              height: MediaQuery.of(context).size.height / 7,
```

Comment:
Estão utilizando o MediaQuery.of(context).size várias vezes neste arquivo, sugiro a criação de uma variável para armazenar esse valor

Reply:
Para ser sincero acho que não é necessário, até porque só foram usadas 3 vezes (concordaria se fosse em outro contexto de variáveis repetidas) em um Widget e em um Center, o que impede (até onde eu saiba) de ser criada uma variável que atenda os dois (necessita do context).

Reply:
Não entendi o comentário, mas sem problemas então